/ptz/axis_camera_tracker:
  ros__parameters:
    aruco_numbers_ugv:
      - -1
    aruco_numbers_uav:
      - -1
    aruco_selfie: 666
    aruco_size: 0.095
    calibration_file_path: /home/neo/workspace/src/axis_camera_driver/src/axis_camera_tracker/config/calibration_9999.yaml
    classes_targets:
      - bottle
      - person
    err_thr: 0.1
    err_thr_zoom: 0.1
    focal_x_m: 1.3832
    focal_x_x0: 300.0
    focal_x_y0: 2986.70
    focal_y_m: 1.4200
    focal_y_x0: 300.0
    focal_y_y0: 2989.88
    frame_camera: ptz/camera_link
    frame_fixed: map
    frame_wall: ptz/wall_link
    frames_track_to_search: 50
    frames_wait_after_selfie: 20
    k_tilt: 0.1
    kpx: 2.0
    kpy: 2.0
    model_score_threshold: 0.5
    n_loops: 2
    padding: 20
    pan_l: -110.0      # pan_l < pan_r
    pan_r: -70.0
    pan_step: 1.0
    pt_path: /home/neo/workspace/logs/yolov8l-seg.pt
    selfie_offset: 0.5
    selfie_save_path: '/home/neo/workspace/logs'
    thr_sharpness: 100.0
    thr_time: 10.0
    tilt_des_down: -30.0  # -45.0
    tilt_des_up: -10.0
    track_aruco_backwards_distance: 1.0
    track_object_forwards_distance: 2.0
    zoom_des: 9999.0
    zoom_num_pics: 10
    zoom_search: 300.0
    zoom_search_step: 300.0
    zoom_selfie_start: 300.0
    zoom_selfie_end: 5000.0
    zoom_step_high: 1500.0
    zoom_step_low: 400.0
